lib/commandCenter/commandCenter.js

```diff
@@ -1,39 +1,39 @@
-var async = require('async');
-var zlib = require('zlib');
-var fs = require('fs');
-var path = require('path');
-var util = require('util');
-var functionArguments = require('function-arguments');
-
-var EventEmitter = require('events').EventEmitter;
-var ResponseCache = require('./responseCache').ResponseCache;
-var httpBatchHandler = require('./httpBatchHandler');
-
-var mage;
-var logger;
-var State;
-
-var DEFAULT_RESPONSE_CACHE_TTL = 3 * 60;  // cache lifetime of the command response in seconds
-var COMPRESSION_THRESHOLD = 4096;         // only gzip if the response is at least this many bytes uncompressed
-var COMMANDS_DIR_NAME = 'usercommands';     // user command directory inside of module file space
-var COMMAND_PATH_SPLITTER = '.';                // what separates the module name and command name
-var CONTENT_TYPE_JSON = 'application/json; charset=UTF-8';
+'use strict';
```

Comment:
Alright, coverage jumped significantly now. The only uncovered bits are those where user commands are pulled from disk. But the logic behind registering the user commands is covered.